### DISCUSSION TITLE
Einweisungsmatrix Spalten tauschen

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#20] [Ideas]
Bei größeren Einrichtungen werden die Seiten bei einem Export teilweise sehr lang.
Idee: Spalten tauschen?

**Comment by moritzvolkmar** [comment]
Weitere Idee: Export der Matrix als PDF

**Comment by YvesR** [comment]
- Kein Export, wir prüfen mal Spalten switch.
- Dazu Filter Button z.B. in "Auswahl anpassen".
